src/redux/actions/componentConfig/fetchPlotDataWork.js

```diff
@@ -9,8 +9,10 @@ import { fetchWork } from 'utils/work/fetchWork';
 
 const getClusterNames = (state) => {
   const clusterIds = state.cellSets.hierarchy[0]?.children.map((entry) => entry.key);
-  const clusterNames = clusterIds?.map((clusterId) => state.cellSets.properties[clusterId].name);
+  const customClusterIds = state.cellSets.hierarchy[1]?.children.map((entry) => entry.key);
```

Comment:
You're hardcoding the position of the cell sets in in the hierarchy (e.g. `0`, `1`) - I don't think you should do this. We want to detect changes to any cell sets in the platform right? So instead of hardcoding the values, I think you can iterate over all the cellsets in hierarchy and then get the name from there? E.g. `state.cellSets.hierarchy.reduce((cellSet) => cellSet...)`